src/main/java/org/fundacionjala/sfdc/pageobjects/common/SFCommonObjects.java

```diff
@@ -1,5 +1,6 @@
 package org.fundacionjala.sfdc.pageobjects.common;
 
+import org.fundacionjala.sfdc.core.driver.DriverManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.core.driver.DriverManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)